examples/Clients/Chatter/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Common;
+using Grpc.Core;
+using Chat;
+
+namespace Sample.Clients
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var name = args[0];
```

Comment:
nit: I think an error message is a less confusing option than a core dump if the sample is called w/o args.